src/freenet/clients/fcp/ClientRequest.java

```diff
@@ -27,6 +27,9 @@
  */
 public abstract class ClientRequest implements Serializable {
 
+    /**
+     * ATTENTION: When incrementing this, please skip version 2. Version 2 had already temporarily
+     * been used by a development branch. */
```

Comment:
OK nice, I therefore filed a bug to implement upload queue backup: https://bugs.freenetproject.org/view.php?id=6533